src/ucp/core/ucp_worker.c

```diff
@@ -2413,6 +2414,26 @@ ucs_status_t ucp_worker_query(ucp_worker_h worker,
         attr->max_am_header = ucp_am_max_header_size(worker);
     }
 
+    if (attr->field_mask & UCP_WORKER_ATTR_FIELD_BANDWIDTH) {
+        attr->bandwidth.dedicated = 0;
+        attr->bandwidth.shared    = 0;
+
+        UCS_BITMAP_FOR_EACH_BIT(context->tl_bitmap, tl_id) {
+            /* If only network devices are requested and this is not a network
+               device - skip it */
+            if ((attr->field_mask & UCP_WORKER_ATTR_FIELD_ADDRESS_FLAGS) &&
+                (attr->address_flags & UCP_WORKER_ADDRESS_FLAG_NET_ONLY) &&
```

Comment:
cache address_flags in local var by using UCP_PARAM_VALUE

Reply:
Can't use `UCP_PARAM_VALUE`, as it requires the field mask enum values to have the form `*_PARAM_FIELD_*`, while ucp_worker_attr_field values are `_ATTR_FIELD`.